client/src/App.jsx

```diff
@@ -21,6 +21,8 @@ import useAuth from './auth/useAuth';
 import { ProfileProvider } from './assets/context/ProfileContext';
 import Network from './assets/components/network/Network';
 import Adminpage from './assets/components/Admin/Adminpage';
+import Usersprofile from './assets/components/Admin/Usersprofile';
+import EditUsersprofile from './assets/components/Admin/EditUsersprofile';
```

Comment:
This is redundant as well, the EditProfile page currently knows when the User or an Admin/Owner roles are viewing and will allow changes to happen. This seems to be a duplicate of Edit Profile and does not need to be added. 

Furthermore, EditProfile is also being redesigned by another contribute in issue #86 

Please remove this.